python/packages/ai/teams/ai/planners/action_planner.py

```diff
@@ -149,6 +158,43 @@ async def complete_prompt(
             template=template,
         )
 
+    def _handle_action_tools(self, memory: MemoryBase, template: PromptTemplate) -> MemoryBase:
+        if not memory.get(SUBMIT_TOOL_OUTPUTS_VARIABLE):
+            memory.set("temp.tool_choice", template.config.completion.tool_choice)
+            memory.set("temp.parallel_tool_calls", template.config.completion.parallel_tool_calls)
```

Comment:
Do these need to be set here if we're resetting the state immediately below?

Reply:
try to call tools - couldn't validate
first time calling tools 

first time setting them to memory (technically could be repeated, see above)